src/main/java/org/folio/uk/controller/AuthUserController.java

```diff
@@ -0,0 +1,46 @@
+package org.folio.uk.controller;
+
+import java.util.UUID;
+import lombok.RequiredArgsConstructor;
+import org.apache.commons.lang3.StringUtils;
+import org.folio.uk.domain.dto.User;
+import org.folio.uk.integration.keycloak.KeycloakService;
+import org.folio.uk.rest.resource.AuthUserApi;
+import org.folio.uk.service.UserService;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequiredArgsConstructor
+public class AuthUserController implements AuthUserApi {
+
+  private final KeycloakService keycloakService;
+  private final UserService userService;
+
+  @Override
+  public ResponseEntity<Void> createAuthUser(UUID userId) {
+    boolean userExists = keycloakService.findKeycloakUserWithUserIdAttr(userId).isPresent();
```

Comment:
We usually put all the logic inside service method: method in controller just call some service method and return a result, like here for example https://github.com/folio-org/mod-users-keycloak/blob/9764693cdc3c026be0f109dbd7077afc07a40bee/src/main/java/org/folio/uk/controller/UserController.java#L26

Please follow this pattern

Reply:
Seems rather unnecessary to have completely hollow controllers. The point of a controller, as the name suggests, is to control the flow. It's not very reasonable to have all flow control in services, and make controller just a proxy for a service.